docs/usage/logging.md

```diff
@@ -10,30 +10,60 @@ Kubernetes uses the underlying container runtime logging, which does not persist
 * One Loki Statefulset in the `garden` namespace which contains logs for the seed cluster and one per shoot namespace which contains logs for shoot's controlplane.
 * One Grafana Deployment in `garden` namespace and two Deployments per shoot namespace (one exposed to the end users and one for the operators). Grafana is the UI component used in the logging stack.
 
+### Extension of the logging stack
+![](images/shoot-node-logging-architecture.png)
+The Logging stack is extended to scrapes logs from the systemd services of each shoot's node and from all gardener components in the shoot `kube-system` namespace. Those logs are exposed only to the gardener operators.
+
 ### How to access the logs
-The first step is to authenticate in front of the Grafana ingress. The secret with the credentials can be found in `garden-<project>` namespace under `<shoot-name>.monitoring`.
-Logs are accessible via Grafana UI. Its URL can be found in the `Logging and Monitoring` section of a cluster in the Gardener Dashboard.
-
-There are two methods to explore logs:
-* The first option is to use the `Explore` view (available at the left side of the screen). It is used for creating log queries using the predefined filters in Loki. For example: 
-`{pod_name='prometheus-0'}`
-or with regex:
-`{pod_name=~'prometheus.+'}`
-
-* The other option is to use `Dashboards` panel. There are custom dashboards for pod logs with one selector field for `pod_name` and one `search` field. The `search` field allows to filter the logs for a particular string. The following dashboards can be used for logs:
-
-  * Garden Grafana
-    * Pod Logs
-    * Extensions
-    * Systemd Logs
-  * User Grafana
-    * Kube Apiserver
-    * Kube Controller Manager
-    * Kube Scheduler
-    * Cluster Autoscaler  * Operator Grafana
-  * Operator Grafana
-    * All user's dashboards
-    * Kubernetes Pods
+The first step is to authenticate in front of the Grafana ingress.
+There are two Grafana instances were the logs are accessible from.
+  1. The user(stakeholder/cluster-owner) Grafana consist of a predefined  `Monitoring and Logging` dashboards which help end-user to get the most important metrics and logs out of the box. This Grafana UI is dedicated only for the end-users and does not show logs rated as sensitives. Also, the `Explore` tab is not available. Those logs are in the predefined dashboard named `Controlplane Logs Dashboard`.
```

Comment:
Why, you don't like 
```text
does not show logs rated as sensitives.
```
which is the real truth why they are not exposed to the end-user.

```text
dashboards which helps end-users
```
dashboards is plural -> help and not helps.
end-user(singular) enclosure all of the users.

Reply:
I think that we do not have filters which decide which log from one container is for user and which for operator.
In my understanding (if the setup is still the same) we just have filters which component's logs to be presented in Operator's Grafana and which to both Grafana instances.
That's why I suggested this message because it describes the behaviour more clearly.

Reply:
Now? 
/ping @Kristian-ZH